tests/phpunit/Civi/Civioffice/PhpWord/PhpWordTemplateProcessorTest.php

```diff
@@ -194,6 +194,69 @@ public function testReplaceSpanMultiple(): void {
     static::assertXmlStringEqualsXmlString($expectedMainPart, $templateProcessor->getMainPart());
   }
 
+  public function testReplaceLineBreak(): void {
+    $mainPart = <<<EOD
+      <w:document xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main">
+        <w:body>
+          <w:p>
+            <w:pPr>
+              <w:pStyle w:val="Normal"/>
+            </w:pPr>
+            <w:r>
+              <w:rPr>
+                <w:b w:val="true"/>
+              </w:rPr>
+              <w:t>Foo {place.holder} bar</w:t>
+            </w:r>
+          </w:p>
+        </w:body>
+      </w:document>
+    EOD;
```

Comment:
Doesn't really matter, but this adds two spaces before each heredoc line, as the closing identifier's indentation defines the number of whitespaces removed from heredoc lines.